components/ActivityIndicator/src/MaterialActivityIndicator.bundle/Resources/en.lproj/MaterialActivityIndicator.strings

```diff
@@ -0,0 +1,8 @@
+/* Accessibility label for indeterminate activity indicator in progress */
+"MaterialActivityIndicatorInProgressAccessibilityLabel" = "In Progress";
```

Comment:
I think the words that aren't the start of each quote should be lowercase. Sentence case.